README.md

```diff
@@ -3,53 +3,62 @@
-
-However, customers looking for any of the following are expected to prefer a setup where the LoRaWAN network servers runs directly on the gateway/Azure IoT Edge:
-
-- Primarily coverage on their own ground (e.g. manufacturing plants, smart buildings, facilities, ports).
-- Capabilities that Azure IoT edge brings to the table:
-  - Local processing on the gateway.
-  - Offline capabilities of the gateway.
-  - Gateway management.
-- Homogenous management of devices and gateways independent of connectivity technology.
-
-## Functionality
+The LoRaWAN starter kit is an *experimental* cross platform private network implementation of the [LoRaWAN specification]('https://lora-alliance.org/resource-hub/lorawantm-specification-v102'). It enables users to setup their own LoRaWAN network that can connect to LoRa based nodes (sensors) and send decoded message packets to Azure IoT Hub for cloud based processing, analytics and other workloads. Alternatively, it allows sending commands from the cloud to the end nodes. The goal of the the project is to provide guidance and a reference for Azure IoT Edge users to experiment with LoRaWAN technology.
+  
+  - [Features](#features)
+  - [LoRaWAN Specification Support](#lorawan-specification-support)
+  - [Prerequisites](#prerequisites)
+  - [Getting Started](#getting-started)
+  - [Known Issues and Limitations](#known-issues-and-limitations)
+  - [Tested Gateways](#tested-gateways)
+  - [Support](#support)
+  - [Contributing](#contributing)
+
+## Features
+- LoRaWAN 1.0.2 implementation (see [LoRaWAN Specification Support](#LoRaWAN-1.0.2-Specification-Support) for more details)
+- Device and Gateway management done completely through Azure IoT Hub.
+- Bi-directional communication between LoRa end devices and Azure cloud.
+- Custom Encoding and decoding framework.
+- Identity Translation for LoRa devices with caching support.
+- Offline and Casually connected Gateways scenarios.
+- Easy deployment and setup using Azure ARM templates.
+- Small to Midsize Scalability Tests.
+- Simulator for development and testing without the need to own a Gateway.
+  
+## LoRaWAN Specification Support
 
+- Current supported Specification: *1.0.2*
 - Support of Class A devices
+- Support of **EU868** and **US915** channel frequencies
 - Activation through ABP and OTAA
 - Confirmed and unconfirmed upstream messages
 - Confirmed and unconfirmed downstream messages
-- Device and Gateway management done completely in Azure IoT Hub
-- Support of EU868 and US915 channel frequencies
-- Experimental support of MAC commands
-- Multi-gateway support
+- Support of MAC commands
+- ADR Support
```

Comment:
ADR is not in there yet, I know that we are working on it but with the rule DEV always shippable it could happen that we do a version still without ADR

Reply:
I am refraining from adding WIP on the ReadMe, basically, the Release Notes should reflect what is our progress and hence I have a comment immediately below. Problem with having WIP / In progress in multiple places of documentation is that it needs to be updated at several places when we make fixes. So, the ReadMe shows all features that "will" be supported but with a note to check out the release notes for currently supported features. do you think this is confusing, in that case I will remove ADR?

Reply:
Yes let's remove ADR until we have something more on this front

Reply:
ok, I thought about this again, and have now started using an "*" against planned features, I don't want to discard ADR completely as we may do it before the end of March. so keeping an asterisk will let people that it may come but no commitments. Following your approach of calling it our in Readme :)

Reply:
Multi gateway is already available
We should also add message de-duplication

Reply:
changed